style.md

```diff
@@ -15,6 +15,56 @@ The redundancy reduction from removing the namespacing prefix is not useful enou
 So we have a consistent way to identify unset values ([source][optional-pointer]).
 The exceptions are entries where the Go default for the type is a no-op in the spec, in which case `omitempty` is sufficient and no pointer is needed (sources [here][no-pointer-for-slices], [here][no-pointer-for-boolean], and [here][pointer-when-updates-require-changes]).
 
+## Examples
+
+### Anchoring
+
+For any given section that provides a notable example, it is ideal to have it denoted with [markdown headers][markdown-headers].
+The level of header should be such that it is a subheader of the header it is an example of.
+
+#### Example
+
+```markdown
+## some topic
+
+### some subheader
+
+#### further subheader
+
+### Example
+
+```
+
+### Content
+
+Where necessary, the values can be exampled as their empty or unset state, but accommodate with comments regarding this intention.
```

Comment:
nit: I don't here “exampled” very often.  How about “… values can be empty or unset, but …”.

Stepping back, it's hard to imagine a case where an empty or unset value would be necessary.  Can you give an example of a setting for which there is no useful example?